.github/workflows/push.yml

```diff
@@ -152,30 +180,63 @@ jobs:
           args: analyse src --configuration phpstan.neon
 
   psalm:
-    runs-on: ubuntu-latest
-    needs: [setup, phpunit]
+    name: Psaml
```

Comment:
little typo here

Reply:
Could you please fix that in your pr? I think that one is ready to merge by the way :+1: